formatTest/typeCheckedTests/expected_output/attributes.re

```diff
@@ -7,11 +8,11 @@
  * This has a nice side effect when printing the terms:
  * If a node has attributes attached to it,
  */
+
```

Comment:
Was this intentionally changed? I had originally thought that all subsequent comments should have no spaces between them. Are you thinking that multiple subsequent single `*` comments should have no new lines but `**` comments should always have separating newlines?